app/src/main/java/lt/vilnius/tvarkau/fragments/BaseReportListFragment.kt

```diff
@@ -52,16 +53,26 @@ abstract class BaseReportListFragment : BaseFragment(), ReportListView {
                 .subscribeOn(ioScheduler)
                 .observeOn(uiScheduler)
                 .subscribe({
-                    reloadData()
+                    reloadReports = true
```

Comment:
While the fix would work, I think it's a hack :)

`onError` handler is needed here.

And unsubscribe should be before `super.onDestroyView()`. That should be enough I think.